cmake/CheckDGtalOptionalDependencies.cmake

```diff
@@ -21,7 +21,7 @@ OPTION(WITH_MAGICK "With GraphicsMagick++." OFF)
 OPTION(WITH_ITK "With Insight Toolkit ITK." OFF)
 OPTION(WITH_CAIRO "With CairoGraphics." OFF)
 OPTION(WITH_HDF5 "With HDF5." OFF)
-OPTION(WITH_QGLVIEWER "With LibQGLViewer for 3D visualization (Qt required)." OFF)
+OPTION(WITH_QGLVIEWER_QT "With LibQGLViewer for 3D visualization (Qt required)." OFF)
```

Comment:
Please keep the WITH_QGLVIEWER flag, this flag is used at many places in the code.